internal/policy/container/has_prohibited_container_name.go

```diff
@@ -0,0 +1,57 @@
+package container
+
+import (
+	"context"
+	"strings"
+
+	"github.com/go-logr/logr"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/image"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/log"
+)
+
+var _ check.Check = &HasProhibitedContainerName{}
+
+type HasProhibitedContainerName struct{}
```

Comment:
Should this have been something like `HasNoTrademarkViolations`?

Reply:
This only covers the container name, so a specific test for that is needed. Maybe if we have more tests that they want lumped together, we can combine.